azure/production/__main__.py

```diff
@@ -160,7 +150,7 @@
                 ),
                 containerinstance.EnvironmentVariableArgs(
                     name="LLAMA_INDEX_WHICH_INDEX",
-                    value="all_data",
```

Comment:
I think this is meant to be `all_data`. Maybe we forgot to switch back from handbook from a test?